wp-media/wp-rocket
# Closes #5085 on RUCSS compatibility with unlimited elements dynamic inline styles

## Description

Added new third party class for unlimited elements plugin to preserve dynamic inline styles in RUCSS

Fixes #5085 

## Type of change


## Is the solution different from the one proposed during the grooming?

This solution is no way different from the proposed.

## How Has This Been Tested?

- Install and activate unlimited elements.
- install any widget from the plugin and use on elementor.
- check page source to see that inline styles are preserved.

# Checklist:
